server/hub.go

```diff
@@ -180,7 +180,7 @@ func (h *Hub) run() {
 					xoriginal: join.pkt.Original,
 					// Indicates a proxy topic.
 					isProxy:   globals.cluster.isRemoteTopic(join.pkt.RcptTo),
-					sessions:  make(map[*Session]perSessionData),
+					sessions:  make(map[*Session]*perSessionData),
```

Comment:
The way I understand memory management in Go, small structs should generally not be stored as pointers.

Reply:
Oh, interesting. Changed back to values.